### PR TITLE
Evaluation profile for event mesh

### PR DESCRIPTION
**Pull request by berkanercan**
**Description**

Changes proposed in this pull request:

- Added evaluation profile resource values to knative-eventing helm chart
- Added evaluation profile resource values to the helm chart of application operation under components/application-operator

These changes can handle two events per second.

*Testing*
A kyma cluster with an external application binded is used to test the resource values. 

**Related issue(s)**
[`See also #9673 `](https://github.com/kyma-project/kyma/issues/9673)

### CODE REVIEW

File: resources/nats-streaming/templates/destination-rule.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.istio.io/v1beta1
+apiVersion: networking.istio.io/v1alpha3
```
Comment (k15r) [inline resources/nats-streaming/templates/destination-rule.yaml:1]: ????

File: resources/nats-streaming/values.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ global:
       # If set to "-", storageClassName: "", which disables dynamic provisioning
       # If defined, storageClassName: <storageClass>
       # storageClass: "-"
-      enabled: true
+      enabled: false
```
Comment (k15r) [inline resources/nats-streaming/values.yaml:31]: why do we disable this?

File: resources/nats-streaming/templates/destination-rule.yaml (hunk 1 above)
Comment (k15r) [inline resources/nats-streaming/templates/destination-rule.yaml:1]: why do we downgrade the apiversion?

**Comment by berkanercan** [comment]
This PR is obsolete.

**Pull request closed without merging**